src/SFA.DAS.Commitments.Support.SubSite/Enums/CommitmentRequestStatus.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.ComponentModel;
+
+namespace SFA.DAS.Commitments.Support.SubSite.Enums
+{
+    public enum RequestStatus
+    {
+        None, // No use here.
+
+        [Description("New request")]
+        NewRequest,
+
+        [Description("Sent to provider")]
+        SentToProvider,
+
+        [Description("Sent for review")]
```

Comment:
There is an open PR that changes the description to 'Under review with employer'.